connector/src/yang/connector/gnmi.py

```diff
@@ -161,140 +179,97 @@ class Gnmi(BaseConnection):
+            proto.gnmi_pb2.PathElem(),
+            proto.gnmi_pb2.PathElem()
+        )
+    >>> path1.name, path2.name, path3.name, path4.name = (
+            'syslog',
+            'messages',
+            'message',
+            'node-name'
+        )
+    >>> path.elem.extend([path1, path2, path3, path4])
+    >>> request.path.append(path)
+    >>> resp = device.gnmi.get(request)
+    >>> print(resp)
     >>> #####################
     >>> # Capabilities      #
     >>> #####################
-    >>> from yang.connector.gnmi import Gnmi
-    >>> kwargs={
-    ... 'host':'172.23.167.122',
-    ... 'port':'50051',
-    ... 'root_certificate':'root.pem',
-    ... 'username':'admin',
-    ... 'password':'C!sco123',
-    ... 'ssl_name_override':'ems.cisco.com'
-    ... }
-    >>> gnmi=Gnmi('iosxe', **kwargs)
-    >>> resp = gnmi.capabilities()
-    >>> #####################
-    >>> # Set example       #
-    >>> #####################
-    >>> content={
-    ... 'namespace': {'ios: 'http://cisco.com/ns/yang/Cisco-IOS-XE-native',
-    ... 'ios-cdp': 'http://cisco.com/ns/yang/Cisco-IOS-XE-cdp'},
-    ... 'nodes': [{'xpath': '/ios:native/ios:cdp/ios-cdp:holdtime',
-    ... 'value': '10'}]
-    ... }
-    >>> resp = gnmi.set(content)
-    >>> #####################
-    >>> # Get mode='CONFIG' #
-    >>> #####################
-    >>> content={
-    ... 'namespace': {'ios: 'http://cisco.com/ns/yang/Cisco-IOS-XE-native',
-    ... 'ios-cdp': 'http://cisco.com/ns/yang/Cisco-IOS-XE-cdp'},
-    ... 'nodes': [{'xpath': '/ios:native/ios:cdp/ios-cdp:holdtime'}]
-    ... }
-    >>> resp = gnmi.get_config({'content': content, 'returns': returns})
-    >>> #####################
-    >>> # Get mode='STATE' #
-    >>> #####################
-    >>> content={
-    ... 'namespace': {'ios: 'http://cisco.com/ns/yang/Cisco-IOS-XE-native',
-    ... 'ios-cdp': 'http://cisco.com/ns/yang/Cisco-IOS-XE-cdp'},
-    ... 'nodes': [{'xpath': '/ios:native/ios:cdp/ios-cdp:holdtime'}]
-    ... }
-    >>> resp = gnmi.get({'content': content, 'returns': returns})
-    >>> #####################
-    >>> # Subscribe example #
-    >>> #####################
-    >>> content={
-    ... 'format': {
-    ...     'encoding': 'JSON',
-    ...     'request_mode': 'STREAM',
-    ...     'sample_interval': 5,
-    ...     'stream_max': 15,
-    ...     'sub_mode': 'SAMPLE'},
-    ... }
-    >>> # Add namespace, node xpath list similar to Get to the content
-    >>> # Add list of expected return values to content
-    >>> # Example of an expected return value:
-    >>> content['returns']=[{
-    ... 'datatype': 'string',
-    ... 'id': 0,
-    ... 'name': 'name',
-    ... 'op': '==',
-    ... 'selected': True,
-    ... 'value': 'v4acl',
-    ... 'xpath': '/acl/acl-sets/acl-set/name'}]
-    >>> # Trigger an event that would kick off subscribe
-    >>> # Call notify_wait() passing in a callback class (see notify_wait)
-
+    >>> resp=device.capabilities()
+    >>> resp.gNMI_version
+    '0.7.0'
```

Comment:
The simple example, capabilities, should be first in the docstring followed by the more complex get example.

Why are you printing only the ``resp.gNMI_version`` of capability response?  Why not print entire resp?

Reply:
Full resp is very large, so i gave and example of only one attribute